serverlessworkflow/modules/ROOT/pages/getting-started/java-embedded-workflows.adoc

```diff
@@ -0,0 +1,104 @@
+            JsonNodeModel result = application.execute(workflow, Collections.emptyMap()); <4>
+            logger.info("Workflow execution result is {}", result.getWorkflowdata()); <5>
+        }
+    }
+}
+----
+<1> Reads the workflow file definition from the project root directory
+<2> Creates a static workflow application object. It is done within the try block since the instance is `Closeable`. This is the reference that allow you to execute workflow definitions.
+<3> Reads the Serverless Workflow Java SDK `Workflow` object from the file. 
+<4> Execute the workflow, passing `Workflow` reference and no parameters (an empty Map). The result of the workflow execution: process instance id and workflow output model, can accessed using `result` variable. 
+<5> Prints the workflow model in the configured standard output. 
+
+If you compile and execute this java class, you will see the following log in your configured standard output
+----
+Workflow execution result is {"greeting":"Hello World","mantra":"Serverless Workflow is awesome!"}
+----
+
+[[embedded-fluent-quick-start]]
+== Hello world (using fluent API)
+
+Using the same Maven setup than in the previous section, you can programatically generate that workflow definition rather than loading it from a file definition by using link:{kogito_runtimes_url}/kogito-serverless-workflow/kogito-serverless-workflow-fluent/src/main/java/org/kie/kogito/serverless/workflow/fluent[fluent API]
+
+Therefore, you can modify the previous example in a way that generates exactly the same output when it is executed, but rather creating a `FileReader` that reads the `Workflow` object, we create the `Workflow` object using java statements. The resulting modified main method is the following
+
+[source,java]
+----
+      try (StaticWorkflowApplication application = StaticWorkflowApplication.create()) {
+            Workflow workflow = workflow("HelloWorld"). <1>
+            		start( <2>
+            				inject( <3>
+            						jsonObject().put("greeting", "Hello World").put("mantra","Serverless Workflow is awesome!"))) <4>
+            		.end() <5>
+            		.build(); <6>
+            logger.info("Workflow execution result is {}",application.execute(workflow, Collections.emptyMap()).getWorkflowdata());  <7>
+        }
+----
+<1> Creates a workflow which name is `HelloWorld`
+<2> Indicate that you are going to specify the start state
+<3> A Inject state is the start state
+<4> Inject state accepts static json, therefore this line creates the JSON data
+<5> End the workflow definition 
+<6> Build the workflow definition
+<7> Execute and print as in previous example
+
+
+== Additional resources
+
+You can find additional and commented examples of fluent API usage (including jq expression evaluation and orchestation of rest services) link:https://github.com/fjtirado/java-flow/tree/main/java-flow-example/src/main/java/org/kie/kogito/serverless/workflow/examples[here]
```

Comment:
Wondering if we should keep this guide small and refer to further, already written examples or we prefer the "verbose" approach, with the risk of outdating

Reply:
Either examples and docs can be outdated. But I guess having examples in our CI, makes them easier to spot and then update.

Reply:
I agree, let me move this classes to our examples repo and link them properly

Reply:
Created https://github.com/apache/incubator-kie-kogito-examples/pull/1895 for adding the examples